### DISCUSSION TITLE
How to add a Github icon?

### DISCUSSION THREAD

**Discussion by leodip** [squidfunk/mkdocs-material#6387] [Q&A] [answered]
Can someone please point me to the directions of how to add a Github icon with a link to my GH repository, in mkdocs material?

It can be at the top-right corner, or in the navbar... 

I couldn't find any example.

Thanks

**Comment by alexvoss** [comment, marked as answer]
Hi, this should help? https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/?h=repository#adding-a-git-repository

**Comment by leodip** [comment]
Amazing, thank you!
